### PR TITLE
1ce Pledge

### PR DESCRIPTION
**Pull request by humantraffic**

### CODE REVIEW

**Comment by dckc** [comment]
Hi. Thanks for your contribution!

But your README.md should be in a folder named after your validator. Please don't change the top level README.md.

**Review by dckc — CHANGES_REQUESTED** [review]
please move your README.md to its own folder

**Comment by dckc** [comment]
11a502a is better, but it deletes the top level README.md

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

File: 1ce/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Agoric Validator Pledge
+
```
[the remark this answers is not in the document]
Comment (humantraffic) [reply 1ce/README.md:3]: add

**Comment by dckc** [comment]
Looks like this PR deleted the top-level readme

**Pull request merged**